docs/dev-guide/integration/rest-api/content/upload.rst

```diff
@@ -94,7 +97,7 @@ The task created will have the following tags.  ``"pulp:repository:<repo_id>",
 :sample_request:`_` ::
 
  {
-    "override_config": {},
+    "override_config": {"mask_id": null},
```

Comment:
mask_id is relevant just for upload of v1 docker images, if you leave it here it will create confusion